src/js/index.js

```diff
@@ -13,6 +13,74 @@ const navBookmark = document.querySelector('[data-js=navBookmark]')
 const navCreate = document.querySelector('[data-js=navCreate]')
 const navSettings = document.querySelector('[data-js=navSettings]')
 
+// Get main sections (^= pages)
+const pages = document.querySelectorAll('.page')
+
+// Get each main section (^= page)
+const createPage = document.querySelector('.create-page')
+const homePage = document.querySelector('.app-grid')
+
+// Get Header Heading
+const headerHeading = document.querySelector('h1')
+
+// Get form
+const form = document.querySelector('form')
+
+// Get textareas
+const questionArea = document.querySelector('.create-page__form__add-question')
+const answerArea = document.querySelector('.create-page__form__add-answer')
+const tagsArea = document.querySelector('.create-page__form__add-tags')
+
+// Get character counters
+const questionCounter = document.querySelector(
+  '.create-page__form__add-question__counter'
+)
+const answerCounter = document.querySelector(
+  '.create-page__form__add-answer__counter'
+)
+const tagCounter = document.querySelector(
+  '.create-page__form__add-tags__counter'
+)
+
+// Get button
+const formButton = document.querySelector('.create-page__form__button')
+
+/* ALPHA TESTING 
+tagsArea.addEventListener('input', () => {
+  return tagCounter.textContent.match(/,/g)
+  ? (tagCounter.textContent = tagCounter.textContent.match(/,/g).length - 5)
+  : (tagCounter.textContent = 5)
+})*/
+
+/* Functions */
+
+// Prevent form from submitting & delete all input
+form.addEventListener('keyup', function (event) {
+  if (event.key === 'Enter') {
```

Comment:
You show quite some knowledge. But this is done by using the Form element and the submit event already. No need for this extra event handling.

Reply:
Ah, I did not think of it being `textarea`s. You are right: without your code here, it would not submit the form on Enter. 

But if we use textareas, we want them to have multiple lines and not submit on Enter – so better get rid of this.